src/bin/bat/assets.rs

```diff
@@ -41,8 +41,14 @@ pub fn assets_from_cache_or_binary() -> Result<HighlightingAssets> {
         }
     }
 
-    Ok(HighlightingAssets::from_cache(&cache_dir)
-        .unwrap_or_else(|_| HighlightingAssets::from_binary()))
+    let no_custom_assets = wild::args_os().any(|arg| arg == "--no-custom-assets");
```

Comment:
It feels wrong to me to parse the entire set of environment arguments again here. It also ignores the possibility that `--no-custom-assets` could be part of `bat`s config file (even if unlikely).

Couldn't we pass a `custom_assets: bool` into this function as a parameter? Make that part of the `Config`, like other settings?

Reply:
I actually went in that direction first, but backed off when I noticed `--no-config` was not in `Config`. But hmm I just now I realized why: `--no-config` is a meta-config, i.e. a config to not use configs :bulb: 

I'll add my new arg to `Config`.